src/Assert.php

```diff
@@ -468,8 +468,8 @@ public static function isAOf($value, $class, $message = '')
         if (!\is_a($value, $class, \is_string($value))) {
             static::reportInvalidArgument(sprintf(
                 $message ?: 'Expected an instance of this class or to this class among his parents %2$s. Got: %s',
-                static::typeToString($value),
-                $class
+                static::valueToString($value),
+                static::valueToString($class)
```

Comment:
Is `static::valueToString` really correct here? 

Shouldn't it always be `'"'.$class.'"'`?

Reply:
`static::valueToString` can be used but it is only used to put it in question marks. I updated it.